Traceback (most recent call last):
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\jupyter_core\utils\__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\ckeulen\anaconda3\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import CoolProp.CoolProp as CP
fluid = "water"  # define the fluid or material of interest

P_1 = 101E+3   #pressure in state (1) in Pa
P_2 = 1E+6   #pressure in state (2) in Pa
m = 1   #water flow-rate in kg/s

h_1 = CP.PropsSI("H", "P", P_1, "Q", 0 , fluid)   #enthalpy in state (1) in J/kg
d_1 = CP.PropsSI("D", "P", P_1, "Q", 0 , fluid)   #density in state (1) in kg/m3
d_2 = d_1   #density in state (2) in kg/m3
h_2 = CP.PropsSI("H", "P", P_2, "D", d_2 , fluid)   #enthalpy in state (2) in J/kg   

W_P = m * (h_2 - h_1)

print('The pump work for the cycle is:', f"{W_P/1000:.1f}", 'kW')
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 1[0m
[1;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mCoolProp[39;00m[38;5;21;01m.[39;00m[38;5;21;01mCoolProp[39;00m [38;5;28;01mas[39;00m [38;5;21;01mCP[39;00m
[0;32m      2[0m fluid [38;5;241m=[39m [38;5;124m"[39m[38;5;124mwater[39m[38;5;124m"[39m  [38;5;66;03m# define the fluid or material of interest[39;00m
[0;32m      4[0m P_1 [38;5;241m=[39m [38;5;241m101E+3[39m   [38;5;66;03m#pressure in state (1) in Pa[39;00m

[1;31mModuleNotFoundError[0m: No module named 'CoolProp'

